### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by emtekaka** [taikii/redmine_custom_auto_complete#3]
Please help
Your plugin is great but works only for the parent task field. When I add a new text field and add a few issues in the project, this new field does not return any values when creating a new issue. What am I doing wrong?

**Comment by taikii** [comment]
Could you check the following 2 settings?

1. Setup the permission for the role
2. Enable module for your project

**Comment by emtekaka** [comment]
Yes, I added the permissions for each role, and activated the appropriate module in the project. There is a magnifier next to the text box and when I enter a value, you can see that something searches but does not show any value.
Maybe I can not quite understand the principle of operation. The search should show all the values that were used in this field in previous issues?

**Comment by taikii** [comment]
To operate auto-complete, you need to type two or more letters.

**Comment by emtekaka** [comment]
Yes of course it starts searching after two or three letters but nothing is find

**Comment by taikii** [comment]
Could you tell me about your Redmine environment? I want to reproduce your problem.

* Redmine version
* Ruby version
* Rails version
* OS kind and version
* Database kind and version
* Other noteworthy matter...

**Comment by emtekaka** [comment]
Thank you so much for wanting to help me. I think the idea of the plugin is fantastic, but I do not know why it works only in that one one field

I ran the project in the docker's container on the Synology server:
Synology OS - DSM 6.2.1-23824 Update 4
Database on MariaDB 10 ver 10.3.7-0051

Those are informations that Redmine gives:

Environment:
  Redmine version                3.3.6.stable
  Ruby version                   2.3.6-p384 (2017-12-14) [x86_64-linux-gnu]
  Rails version                  4.2.7.1
  Environment                    production
  Database adapter               Mysql2
SCM:
  Subversion                     1.8.8
  Darcs                          2.8.4
  Mercurial                      2.8.2
  Cvs                            1.12.13
  Bazaar                         2.7.0
  Git                            2.16.1
  Filesystem                     
Redmine plugins:
  redmine_custom_auto_complete   0.0.1

**Comment by taikii** [comment]
I confirmed it to your environment as much as possible but it worked normally.

* Redmine version 3.3.6.stable
* Ruby version 2.2.10-p489 (2018-03-28) [x86_64-linux]
* Rails version 4.2.7.1
* Database adapter Mysql2
* 10.3.7-MariaDB

Using [MariaDB Docker Image](https://hub.docker.com/_/mariadb/) and [Redmine Docker Image](https://hub.docker.com/_/redmine/).

![screenshot](https://user-images.githubusercontent.com/12361395/51323592-fb56a480-1aab-11e9-9454-a054a5051ce5.gif)

Sorry, I may not have the ability to solve your problem... 😢

**Comment by emtekaka** [comment]
I can not give up, especially if it works for you.
Can you write to me what to do step by step to check the operation?
Maybe I change some small thing and that's why it does not work.
I can install the MariaDB and Redmine docker from the beginning to check everything.

In my case the plugin is looking for and nothing is found. Maybe the fields in the base are called differently? Maybe you could write to me what I should check

**Comment by taikii** [comment]
This is the docker-compose.yml I used for testing. I hope it will be of your help.

https://gist.github.com/taikii/6f22d14455afc79e044f6606d499b076

**Comment by ChetWang** [comment]
I'am still confued.... could you please provide a more detailed decription on how to used this plugin?

**Comment by taikii** [comment]
@ChetWang This plugin just adds autocomplete to short text custom fields.

**Comment by luizjr** [comment]
@taikii Friend will this plugin support version 5 of Redmine?

**Comment by taikii** [comment]
@luizjr I checked in the following environment.
https://gist.github.com/taikii/40d86cdb1e527e120f68f52594c2f407
